crates/anstyle-svg/src/lib.rs

```diff
@@ -97,6 +99,12 @@ impl Term {
         self
     }
 
+    /// Whether or not to generate HTML5 compatible tags.
+    pub const fn use_html5(mut self, use_html5: bool) -> Self {
```

Comment:
What is the prevelance of html5 for whether we should just make this non-configurable?

Reply:
HTML5 is basically used everywhere nowadays. I'm not sure if formats like `epub` are fully-compliant though. So if you want to keep this crate useable by a maximum number of users, I'd say keep the possibility to disable HTML5. We can however discuss about whether or not it should be enabled or disabled by default.

Reply:
In that case, I think I'd prefer we switch over to HTML 5 and we can add an opt-out if someone has a need.

Based on usage, HTML 5 seems like the right choice.  At that point, rather than speculatively have an opt-out, I'd prefer doing it when requested.